src/camelize.ts

```diff
@@ -0,0 +1 @@
+export const camelize = (input: string): string => input.replace(/-([a-z])/g, function (g) { return g[1].toUpperCase(); });
```

Comment:
NIT: Can we write it with an arrow function?
```typescript
input.replace(/-([a-z])/g, g => g[1].toUpperCase())
```

Reply:
done